<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexWidget</class>
 <widget class="QWidget" name="IndexWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="indexGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Search index:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLineEdit" name="indexEdit"/>
      </item>
      <item>
       <widget class="QListWidget" name="indexList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionHideEntry">
   <property name="text">
    <string>Hide entry</string>
   </property>
  </action>
  <action name="actionOriginalOrder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>From file</string>
   </property>
  </action>
  <action name="actionAlphabeticOrder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alphabetic word by word</string>
   </property>
  </action>
  <action name="actionAtergoOrder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>A tergo</string>
   </property>
  </action>
  <action name="actionEditEntry">
   <property name="text">
    <string>Edit entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionViewHidden">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View hidden entries</string>
   </property>
  </action>
  <action name="actionShowEntry">
   <property name="text">
    <string>Show entry</string>
   </property>
  </action>
  <action name="actionLetterOrder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alphabetic letter by letter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
